docs/src/pages/[platform]/getting-started/troubleshooting/index.page.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Troubleshooting
+metaTitle: Troubleshooting
+metaDescription: Troubleshooting Guide - Common issues encountered, with workarounds, when installing Amplify UI.
```

Comment:
Any wordsmiths? 👀

Reply:
Maybe something like: "Workarounds for common issues with installing Amplify UI."

Also, should there be a `description` in addition to the `metaDescription`?

Reply:
To expand on Joe's wordsmithing: 
"Troubleshooting Guide - Workarounds for common issues using Amplify UI."

Reply:
`description` shows on on the page and is the default for `meta`, but if we want do override it as something different, we need to add a `metaDescription` .

In this case, since we are not adding anything visible on the page, `metaDescription` should be good

Reply:
I updated the text and included it as a 'description' because I think it looks nicer having a bit of intro text to describe the page before jumping into the troubleshooting topics (see screenshot  below).

<img width="828" alt="Screen Shot 2022-05-23 at 3 31 42 PM" src="https://user-images.githubusercontent.com/376920/169892753-522dfac5-5173-42ad-8910-f2eaaadb5490.png">